src/components/App.tsx

```diff
@@ -37,6 +40,9 @@ class App extends React.Component<Record<string, unknown>, AppState> {
   }
 
   public render() {
+    // TODO: fetch real ETH price
+    const ethPriceUsd = 2700;
+    this.fetchAave(this.mainAccount, ethPriceUsd);
```

Comment:
unless you want this to run on every render of the DOM you could move this to either
`componentDidMount` or other lifecycle methods here: https://reactjs.org/docs/react-component.html#updating